src/components/studies/participants/modify/EditParticipantForm.tsx

```diff
@@ -123,18 +127,25 @@ const EditParticipantForm: FunctionComponent<EditParticipantFormProps> = ({
                 }></DatePicker>
             ))}
           </>
-
-          <FormControl>
-            <SimpleTextLabel htmlFor="note">Notes</SimpleTextLabel>
-            <SimpleTextInput
-              value={note}
-              placeholder="comments"
-              onChange={e => setNotes(e.target.value)}
-              id="note"
-              multiline={true}
-              rows={5}
+          <Box width="375px" mb={3}>
```

Comment:
Do we need to set the width?

Reply:
Yea, so if there is no set width then the dropdown would span the entire length of the modal: 
![Screen Shot 2021-09-03 at 2 55 20 PM](https://user-images.githubusercontent.com/31671708/132068711-cdcd1308-a748-4f13-9e7c-fa5f3412dd5a.png)